Comp2023/src/main/java/frc/robot/commands/AutoPreloadAndEngageChargeStation.java

```diff
@@ -7,35 +7,48 @@
 import edu.wpi.first.wpilibj2.command.PrintCommand;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.Constants.GRConsts;
+import frc.robot.subsystems.Elbow;
+import frc.robot.subsystems.Gripper;
 import frc.robot.subsystems.Swerve;
+import frc.robot.subsystems.Wrist;
 
 /**
  *
  */
 public class AutoPreloadAndEngageChargeStation extends SequentialCommandGroup
 {
-  public AutoPreloadAndEngageChargeStation(Swerve swerve)
+  public AutoPreloadAndEngageChargeStation(Swerve swerve, Gripper gripper, Elbow elbow, Wrist wrist)
   {
-    setName("AutoPreloadAndEngageChargeStation");
+    setName("AutoPreloadAndLeaveCommunity");
 
     addCommands(
         // Add Commands here:
 
         // @formatter:off
-        new PrintCommand(getName() + ": AUTO PATH SEQUENCE: Run first path"),
-        /* TODO: DRIVE BACKWARD + PRELOAD PATHS / COMMANDS
+        new PrintCommand("AUTO PATH SEQUENCE: Run first path"),
+        new PrintCommand("AUTO: Move Arm for Preload"),        
         new ParallelDeadlineGroup(
-          new WaitUntilCommand(swerve::driveWithPathFollowerIsFinished),
-          new AutoDrivePath (swerve, "driveOffCommunity", true)
+            new ArmSetHeightScoreHigh(elbow, wrist)
         ),
-        */
-        new PrintCommand(getName() + ": AUTO: Run second path"),
+        new PrintCommand("AUTO: Gripper Score"),
         new ParallelDeadlineGroup(
-          new WaitUntilCommand(swerve::driveWithPathFollowerIsFinished),
-          new AutoDrivePath ( swerve, "driveOffCommunity", false)
+          new GripperRun(gripper, GRConsts.GRMode.GR_EXPEL).withTimeout(1)
+        ),
+        new ParallelDeadlineGroup(
+          new GripperRun(gripper, GRConsts.GRMode.GR_STOP)
+          ),
+
+        new PrintCommand("AUTO: Move Arm Down"),
+        new ParallelDeadlineGroup(
+          new ArmSetHeightStow(elbow, wrist)
+        ),
+        new PrintCommand("AUTO: Run to ChargeStation"),
+        new ParallelDeadlineGroup(
+          new AutoChargeStation(swerve)
         ),
 
-        new PrintCommand(getName() + ": AUTO: Hold in place"),
+        new PrintCommand("AUTO: Hold in place"),

```

Comment:
Please use the "getName" on every Print command so it clearly identifies which command is doing the print.